layouts/partials/partnerships.html

```diff
@@ -3,16 +3,9 @@
 {{ if $data.partnerships.partnerships.enable }}
 {{ with $data.partnerships.partnerships }}
 <div>
-    <div class="mt-9 text-center sm:mt-11 lg:mt-14">
-        <h3 class="font-bold text-xl sm:text-2xl lg:text-3xl">
-            <b class="text-[#1D65A6]">{{ with .title }} {{ index (split . " ") 0 | safeHTML }} {{ end }}</b>
-            {{ with .title }} {{ after (len (index (split . " ") 0)) . | safeHTML }} {{ end }}
-        </h3>
-      </div>
-    
-      <ul class="flex mt-10 mx-auto justify-center space-x-20">
+      <ul class="sm:flex mt-10 mx-auto justify-center sm:space-x-24">
         {{ range .partner_item}}
-            <li class="text-center">
+            <li class="p-4 sm:p-0 text-center">
```

Comment:
`p-4` was added here to add a little space between the logos when they're on screens smaller than 640px. 
`sm:p-0` was added to remove the padding since it will not be needed on screens with a width of 640px or wider.